daemon/logger/jsonfilelog/read.go

```diff
@@ -65,18 +65,22 @@ func (l *JSONFileLogger) readLogs(logWatcher *logger.LogWatcher, config logger.R
 	}
 	defer latestFile.Close()
 
-	files = append(files, latestFile)
-	tailer := ioutils.MultiReadSeeker(files...)
-
 	if config.Tail != 0 {
+		moreFiles := append([]io.ReadSeeker{latestFile}, files...)
```

Comment:
I think it's better to pass append directly to MultiReadSeeker. Also, you changed an order of slice, does that fix something?